blog/models.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Post(models.Model):
+    owner = models.ForeignKey("User", on_delete=models.CASCADE)
```

Comment:
The ForeignKey reference to 'User' should use the AUTH_USER_MODEL setting instead of a hardcoded string. This ensures compatibility with custom user models.